src/plugins/d3/renderer/hooks/useShapes/scatter/index.tsx

```diff
@@ -48,17 +56,32 @@ export function ScatterSeriesShape(props: ScatterSeriesShapeProps) {
         const inactiveOptions = get(seriesOptions, 'scatter.states.inactive');
 
         const selection = svgElement
-            .selectAll('circle')
+            .selectAll('path')
             .data(preparedData, shapeKey)
             .join(
-                (enter) => enter.append('circle').attr('class', b('point')),
+                (enter) => enter.append('path').attr('class', b('point')),
                 (update) => update,
                 (exit) => exit.remove(),
             )
-            .attr('fill', (d) => d.data.color || d.series.color || '')
-            .attr('r', (d) => d.data.radius || DEFAULT_SCATTER_POINT_RADIUS)
-            .attr('cx', (d) => d.cx)
-            .attr('cy', (d) => d.cy);
+            .attr('d', (d) => {
+                const seriesIdIndex = d.series.index;
+
+                const scatterStyle =
+                    (d.series as ScatterSeries).symbol || getScatterStyle(seriesIdIndex);
+                const scatterSymbol = getScatterSymbol(scatterStyle);
+
+                const size = d.data.radius ? d.data.radius * 2 : DEFAULT_SCATTER_POINT_SIZE;
```

Comment:
This should be defined earlier - here you need to use the already prepared values